README.md

```diff
@@ -5,4 +5,4 @@
 - Lucas Knezevich
 - Jason Beutler
 - Victor Navarro
-- Ashray Thapa
```

Comment:
Not sure if we need a PR since it looks like your name is already in there? But could do another one for more practice

Reply:
Ahh I might have read something wrong. Anyways, you can use the web editor to solve the merge conflict, and then it should be good to go.

Reply:
Thanks that's good to know. But these looked like: 
NAMBOB>>>>>
=========
LASTOMAS>>>>

Have you seen that before?

Reply:
Not a lit like above. Just an equal number of dashes on each side.